source/database-connections/save-mongodb-connection.txt

```diff
@@ -0,0 +1,106 @@
+      :manual:`connection string <mongodb-uri>`.
+
+      If you are using an on-premises deployment, you must use the MongoDB
+      connection string.
+
+      .. tabs::
+
+         .. tab:: Select a cluster
+            :tabid: select-atlas-cluster
+            
+            a. Select the cluster.
+            
+               Clusters are displayed in a three-level hierarchy: 
+               :guilabel:`Organization` > :guilabel:`Project` >
+               :guilabel:`Cluster`, organized alphabetically. Only the first 100 clusters you are authorized to access are shown.
+
+            #. If it isn't included in the connection string, enter the
+               :guilabel:`Database` to connect to.
+            
+            #. If they aren't included in the connection string, enter the 
+               :guilabel:`Username` and :guilabel:`Password` of your
+               :ref:`Relational Migrator MongoDB user <rm-mongodb-service-user>`.
+               
+               Checking :guilabel:`Save password` saves the password securely
+               on your machine. Relational Migrator automatically fills in the 
+               :guilabel:`Password` field when updating connection information.
+
+            If you leave any of the form fields for :guilabel:`Database`,
+            :guilabel:`Username`, or :guilabel:`Password` blank, Relational
+            Migrator uses the values from the Atlas cluster metadata.
+
+         .. tab:: Enter MongoDB URI
+            :tabid: enter-mongodb-uri
+
+            a. In the :guilabel:`MongoDB connection string (URI)` field, input
+               your :manual:`MongoDB URI <mongodb-uri>`.
+
+            #. If it isn't included in the connection string, enter the
+               :guilabel:`Database` to connect to.
+            
+            #. If they aren't included in the connection string, enter the 
+               :guilabel:`Username` and :guilabel:`Password` of your
+               :ref:`Relational Migrator MongoDB user <rm-mongodb-service-user>`.
+               
+               Checking :guilabel:`Save password` saves the password securely
+               on your machine. Relational Migrator automatically fills in the 
+               :guilabel:`Password` field when updating connection information.
+            
+            If you leave the form :guilabel:`Database`, :guilabel:`Username`, 
+            or :guilabel:`Password` fields blank, Relational Migrator uses the
+            values from the URI.
+
+   .. step:: Specify a :guilabel:`Connection name` and optional :guilabel:`Environment tag`
+         
+      Environment tags color code and organize connections for 
+      convenience. Leaving the drop-down empty is the same as selecting 
+      the "None" tag.
+
+   .. step:: (Optional) Click :guilabel:`Test connection`
+
+      Relational Migrator attempts to authenticate and connect to the
+      database.
+         
+   .. step:: Click :guilabel:`Save`
```

Comment:
As a final note can we emphasise that the connection will be available to select and use in this and other projects.